app/controllers/application_controller.rb

```diff
@@ -32,7 +32,7 @@ def intake_from_completed_session
   end
 
   def current_stimulus_triage
-    StimulusTriage.find_by_id(session[:stimulus_triage_id])
+    StimulusTriage.find_by_id(session[:stimulus_triage_id]) unless hub?
```

Comment:
I noticed that this was getting called needlessly in the hub, and experimented with adding an "unless hub?" ... i could also do this with current_intake, or i could remove this one too.

Reply:
I think this is OK to add to this PR.

I would like to see further refactors, but they're not essential. (For example, do we even really still need this?)

Reply:
@alaroia-cfa I don't think so, but if we're going to remove it we also need to remove all of the accompanying stimulus triage flow code that relies on this which, unfortunately, didn't seem isolated enough to just take care of as a consequence of this code. I'll make a ticket for this code removal.

Ultimately, I'd like to do a larger refactor where we isolate controllers for the hub and intake a bit more so that we're not trying to load current intake in the hub, etc.